### ISSUE TITLE
Create tempalte role and role init.

### ISSUE THREAD

**Issue by ypid** [debops/debops-playbooks#278]
With all the format enhancements going on, I thing there should be a easy to use role template for new roles which follows the latest standards. Currently, I maintain my own role template https://github.com/ypid/ypid-ansible-common/tree/master/template_role plus script https://github.com/ypid/ypid-ansible-common/blob/master/bin/ansible-copy-template-role.

I think it could be moved to `debops/ansible-template`.

**Comment by drybjed** [comment]
Good idea. :+1: I wonder if the `template` namespace could be ever useful, any thoughts about `debops/ansible-template_role` instead? The `debops/ansible-template` sounds fine, I just wonder if future-proofing could be useful later on.

**Comment by ypid** [comment]
In this case, I would go with `debops/ansible-template_role` to have matching name spaces. Should also avoid problems with a real role with the name template should ever be added.

**Comment by drybjed** [comment]
Great. :)
